index.html

```diff
@@ -1,92 +1,170 @@
 <!DOCTYPE html>
 <html lang="en">
 
+
 <head>
   <meta charset="UTF-8">
-  <title>Fork Me! FCC: Test Suite Template</title>
-  <link rel="stylesheet" href="css/style.css">
-  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js" defer></script>
-</head>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Dr. Favaloro</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./css/style.css">
 
-<body>
-
-  <!DOCTYPE html>
-  <html lang="en">
+</head>
 
-  <head>
+<header>
+  <div id="main">
+    <h1 id="title"> Dr. René Favaloro </h1>
+    <p>The man who saved a billion lives</p>
+  </div>
+</header>
 
```

Comment:
<header > added group to <div>

Reply:
- [ ] Kindly use comments politely, as suggestions instead of orders, and make sure to ask the `why` question. Why are you requesting these changes?